ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -601,20 +602,14 @@ exampleNewEpochState value = SL.NewEpochState {
                  (keyToCredential examplePayKey)
                  (SL.StakeRefBase (keyToCredential exampleStakeKey))
 
-    rewardUpdate :: SL.RewardUpdate (EraCrypto era)
-    rewardUpdate = SL.RewardUpdate {
-          deltaT    = SL.DeltaCoin 10
-        , deltaR    = SL.DeltaCoin (- 100)
-        , rs        = Map.singleton
-                        (keyToCredential exampleStakeKey) $
-                        Set.singleton $ SL.Reward {
-                            SL.rewardType   = SL.MemberReward
-                          , SL.rewardPool   = (SL._poolId examplePoolParams)
-                          , SL.rewardAmount = SL.Coin 10
-                          }
-        , deltaF    = SL.DeltaCoin (- 3)
-        , nonMyopic = nonMyopic
-        }
+    rewardUpdate :: SL.PulsingRewUpdate (EraCrypto era)
```

Comment:
@jared Do these values need to change?

note to self: Since they changed, is there a golden test that should also change?